pkgs/by-name/pi/picom-pijulius/package.nix

```
@@ -0,0 +1,27 @@
+{
+  picom,
+  lib,
+  fetchFromGitHub,
+  pcre
+}:
+picom.overrideAttrs (
+  finalAttrs: previousAttrs: {
```

we can drop this since it's not used here.

```suggestion
  previousAttrs: {
```

---

No you can't...
You can change it to _: though

---

https://nixos.org/manual/nixpkgs/unstable/#sec-pkg-overrideAttrs
> If only a one-argument function is written, the argument has the meaning of `previousAttrs`.

---

I'm an idiot and was getting the order backwards